### PR TITLE
Separated ops

### PR DESCRIPTION
**Pull request by SoluMilken**
separated pad ops into 
(1) op: add_sos_eos
(2) op: pad

### CODE REVIEW

File: strpipe/ops/tests/test_add_sos_eos.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    ]
+    assert tx_info == [
+        [True, True],
+        [True, True],
+        [True, True],
+    ]
+
+
+def test_transform_wo_sos():
+    op = AddSosEos(sos_token=DefaultTokens.nul)
```
Comment (stegben) [inline strpipe/ops/tests/test_add_sos_eos.py]: Better way is to use `None` or null pointer to determine whether `sos` is required.

File: strpipe/ops/tests/test_add_sos_eos.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    ]
+    assert tx_info == [
+        [False, True],
+        [False, True],
+        [False, True],
+    ]
+
+
+def test_transform_wo_eos():
+    op = AddSosEos(eos_token=DefaultTokens.nul)
```
Comment (stegben) [inline strpipe/ops/tests/test_add_sos_eos.py]: Better way is to use `None` (for Py) or null pointer (for C) to determine whether `eos` is required.

File: strpipe/ops/tests/test_add_sos_eos.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    ]
+    assert tx_info == [
+        [False, False],
+        [False, False],
+        [False, False],
+    ]
+
+
+def test_inverse_transform():
+    op = AddSosEos(eos_token=DefaultTokens.nul)
```
Comment (stegben) [inline strpipe/ops/tests/test_add_sos_eos.py]: Test cases should cover all situations:
- only sos
- only eos
- required both
- required not both

File: strpipe/ops/tests/test_add_sos_eos.py (hunk 1 above)
Comment (SoluMilken) [reply strpipe/ops/tests/test_add_sos_eos.py]: c041906

File: strpipe/ops/tests/test_add_sos_eos.py (hunk 2 above)
Comment (SoluMilken) [reply strpipe/ops/tests/test_add_sos_eos.py]: c041906

File: strpipe/ops/tests/test_add_sos_eos.py (hunk 3 above)
Comment (SoluMilken) [reply strpipe/ops/tests/test_add_sos_eos.py]: c041906

**Pull request merged**